src/calculateRentalCost.js

```diff
@@ -29,6 +29,16 @@
 
 function calculateRentalCost(days) {
   // write code here
+
+  if (days >= 3 && days < 7) {
+    return (40 * days) - 20;
+  }
+
+  if (days >= 7) {
+    return (40 * days) - 50;
+  }
+
+  return 40 * days;
```

Comment:
[checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 1st item, I don't know what 40, 50, 20, 7 and 3 are